src/main/groovy/net/nemerosa/versioning/support/TagSupport.groovy

```diff
@@ -4,14 +4,14 @@ import java.util.regex.Matcher
 
 class TagSupport {
 
-    static int tagOrder(String tagPattern, String tagName) {
+    static BigInteger tagOrder(String tagPattern, String tagName) {
```

Comment:
Hi, do you have a use case where the number of tags would be greater than `Integer.MAX_VALUE` ?